bin/tachyon-mount.sh

```diff
@@ -213,6 +213,7 @@ case "${1}" in
         mount_local $1
         ;;
       workers)
+        bin=`cd "$( dirname "$0" )"; pwd`
         $LAUNCHER $bin/tachyon-workers.sh $bin/tachyon-mount.sh $1
```

Comment:
Sure I'll apply. Thanks for the doc. I'll fix this code as you suggested and create a new [SMALLFIX] PR for the improvements on the other files. Regards.